src/Pos/Ssc/GodTossing/Toss/Pure.hs

```diff
@@ -37,10 +35,10 @@ newtype PureToss a = PureToss
     { getPureToss :: NamedPureLogger (RWS MultiRichmenStake () GtGlobalState) a
     } deriving (Functor, Applicative, Monad, CanLog, HasLoggerName)
 
-instance (Monad m, HasLoggerName m) => HasLoggerName (IdentityT a m) where
-    getLoggerName = lift getLoggerName
+instance HasLoggerName Identity where
```

Comment:
@ChShersh I'll move this to `log-warper` if I also end up needing it in `PurePoll` for CSL-979.